conjure-java-jackson-serialization/build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
     api "com.palantir.safe-logging:preconditions"
     implementation "com.palantir.safe-logging:logger"
     implementation 'com.palantir.tritium:tritium-registry'
+    implementation 'com.github.ben-manes.caffeine:caffeine'
```

Comment:
I think we're probably ok to add this as a dependency given it is already used in many places. If/when there are any major version bumps/API changes those may be tougher to roll out.